src/features/plan/Plan.js

```diff
@@ -1,70 +1,254 @@
-                            badge={
-                                <Badge colorScheme="orange">
-                                    Work in Progress
-                                </Badge>
-                            }
-                        >
-                            <ExpenseList />
+                    </GridItem>
+                    <GridItem>
+                        <Card heading="Progress (Time)">
+                            <Card isNested>
+                                <AspectRatio ratio={16 / 9}>
+                                    <CircularProgressbar
+                                        value={progress}
+                                        text={`${progress}% complete`}
+                                        styles={buildStyles({
+                                            pathColor:
+                                                progress > 100
+                                                    ? "#DC2626"
+                                                    : "#3b82f6",
+                                            trailColor: "#f5f5f5",
+                                            textColor:
+                                                progress > 100
+                                                    ? "#DC2626"
+                                                    : "#3b82f6",
+                                            textSize: "12px",
+                                        })}
+                                    />
+                                </AspectRatio>
+                            </Card>
                         </Card>
-                        <Card
-                            heading="Add Expense"
-                            badge={
-                                <Badge colorScheme="orange">
-                                    Work in Progress
-                                </Badge>
-                            }
-                        >
-                            <AddExpenseForm />
+                    </GridItem>
+                    <Card heading="Progress (Budget)">
+                        <Card isNested>
+                            <AspectRatio ratio={16 / 9}>
+                                <CircularProgressbar
+                                    value={percentage}
+                                    text={`${percentage}% spent`}
+                                    styles={buildStyles({
+                                        pathColor:
+                                            percentage > 100
+                                                ? "#DC2626"
+                                                : "#3b82f6",
+                                        trailColor: "#f5f5f5",
+                                        textColor:
+                                            percentage > 100
+                                                ? "#DC2626"
+                                                : "#3b82f6",
```

Comment:
The lack of equality here is ok. Though I would add a "near exceeding budget" threshold of 90% where it would display in yellow.